NEWS.md

```diff
@@ -1,3 +1,12 @@
+# 4.3.0
+
+### Features
+
+* Add `have_rich_text` matcher for `ActionText` ([#1263])
```

Comment:
Do issue numbers get autolinked in Markdown documents? If so then you can remove the `[...]`, but if not, what do you think about adding a URL for this, a la:

```
[#1106]: https://github.com/thoughtbot/shoulda-matchers/pull/1106
```

Reply:
Good point. They only get linked auto-linked in GitHub, so I added the links so it works anywhere else.